beetsplug/duplicates.py

```diff
@@ -253,13 +253,12 @@ def _order(self, objs, tiebreak=None):
         "completeness" (objects with more non-null fields come first)
         and Albums are ordered by their track count.
         """
-        if tiebreak:
-            kind = 'items' if all(isinstance(o, Item)
-                                  for o in objs) else 'albums'
+        kind = 'items' if all(isinstance(o, Item) for o in objs) else 'albums'
+
+        if tiebreak and kind in tiebreak.keys():
             key = lambda x: tuple(getattr(x, k) for k in tiebreak[kind])
         else:
-            kind = Item if all(isinstance(o, Item) for o in objs) else Album
```

Comment:
Line 269, below, assumes that `kind` is a Model class, not a string.

Reply:
Oops... that code is never run in my specific configuration, but line 269 just needs Item.all_keys(), so it's an easy change.